src/client/viewSandbox.js

```diff
@@ -315,6 +315,10 @@ const init = () => {
         markAsDirty,
       });
 
+      // We expose extensionBridgePromise on window so consumers using the sandbox
+      // for automated testing can more easily exercise the extension view.
+      window.extensionBridgePromise = bridge.promise;
+
```

Comment:
I'm not sure how I feel generally about the sandbox being used for automated testing. It seems like a nice way to functionally test an extension and it prevents the extension developer from having to develop their own sandbox-like testing area. Anyway, exposing the bridge allows the automated tests to execute `init`, `getSettings`, and `validate` on the extension without having to interact with our UI, dig values out of CodeMirror, etc.